lib/core/network/models/user_profile_data.dart

```diff
@@ -10,13 +10,13 @@ class UserProfileData with _$UserProfileData {
 
   factory UserProfileData({
     required String accountName,
-    required Network network,
```

Comment:
@gguijarro-c-chwy is there a way to make the network required, but provide a default when reading old data from JSON?

I didn't see an easy way to do this with the freezed classes. 

That's what I ideally want. I want it to be required, since we need it, but for backwards compatibility, we must provide a default value.

Reply:
I guess I could create a new method fromJSONLegacy to create this object, and custom parse it?!

Reply:
Merging this now so I can ship but will get back to it